edifact-edi/2.3/modules/ROOT/pages/edifact-edi-connector-batching.adoc

```diff
@@ -0,0 +1,60 @@
+= EDIFACT EDI Connector 2.3 - Batch Outbound EDIFACT Messages - Mule 4
+
+For B2B and EDI messages that are not time-sensitive, such as invoices and payment remittances, batch messages instead of transmitting transactions to your trading partners individually.
```

Comment:
Please rephrase as below:

For EDI messages that are not time-sensitive, such as invoice and payment remittances, you can use the Batch operation to merge multiple individual messages into a single interchange and send to your trading partners.